blaze-client/src/main/scala/org/http4s/client/blaze/BlazeClient.scala

```diff
@@ -13,18 +15,11 @@ import scalaz.{-\/, \/-}
 object BlazeClient {
   private[this] val logger = getLogger
 
-  def apply[A <: BlazeConnection](manager: ConnectionManager[A], config: BlazeClientConfig): Client = {
-
-    val shutdownTask = manager.shutdown().flatMap(_ => Task.delay {
-      // shutdown executor services that have been implicitly created for us
-      config.executor match {
-        case es: DefaultExecutorService =>
-          logger.info(s"Shutting down default ExecutorService: $es")
-          es.shutdown()
+  def apply[A <: BlazeConnection](manager: ConnectionManager[A],
```

Comment:
Thats a good point about the manager. I'm happy to leave it open if we require the manager shutdown to be a part of the shutdown Task but also open to making it private. I left it open for the off chance that someone wants to make a custom `ConnectionManager` but that seems pretty uncommon so I can go either way.